src/server/scan/listener/handle-scan-result.js

```diff
@@ -22,15 +22,24 @@ async function handleScanResult(message, scanResultQueueUrl, server) {
   const uploadDetails = await server.redis.findUploadDetails(uploadId)
   const fileDetails = await server.redis.findFileDetails(fileId)
 
+  const childLogger = server.logger.child({
+    payLoadKey: payload.key,
+    uploadId,
+    //  uploadStatus: uploadDetails.uploadStatus,
```

Comment:
can we not commit commented out code please?

Reply:
I got those removed locally, just did not want to do another tiny commit. Assumed there would be other changes to loop them in with :)